eras/shelley/impl/cddl-files/shelley.cddl

```diff
@@ -25,6 +25,10 @@ header_body =
   ]
 
 
+block_number = uint
+
+slot = uint
```

Comment:
Slot is a 64bit word and has always been. So, this is not accurate.
```suggestion
slot = uint .size 8
```
For this reason, I'd recommend moving its definition into `cardano-ledger-core` and reusing it throughout all eras.